README.md

```diff
@@ -96,6 +96,7 @@ Statuspage | Not Vulnerable | `Status page pushed a DNS verification in order to
 Strikingly                           | Vulnerable     | `page not found`                                                 |[Issue #58](https://github.com/EdOverflow/can-i-take-over-xyz/issues/58) | [Strikingly-Sub-takeover](https://medium.com/@sherif0x00/takeover-subdomains-pointing-to-strikingly-5e67df80cdfd)
 Surge.sh                         | Vulnerable     | `project not found`                                                     || [Surge Documentation](https://surge.sh/help/adding-a-custom-domain)
 SurveySparrow | Vulnerable | 'Ouch! Account not found' | [Issue #281](https://github.com/EdOverflow/can-i-take-over-xyz/issues/281) |[Custom domain]( https://help.surveysparrow.com/custom-domain)
+Tave | Not Vulnerable | `  Error 404: Page Not Found` | [Issue #323](https://github.com/EdOverflow/can-i-take-over-xyz/issues/323)
```

Comment:
Is there an actual space at the beginning of the error message?

Reply:
Hello! 

No space, just a mistake.  

[Source](https://github.com/Ice3man543/SubOver/blob/3d258e254ab5b5f37ec6fc2dcd55b45f266ac5f6/providers.json)